src/app/forms/services/dynamic-form.types.ts

```diff
@@ -111,8 +111,10 @@ export interface FormNode {
   isoDate?: boolean;
   class?: string;
   customId?: string;
-  warning?: string;
+  warning?: ValidationErrors | null;
```

Comment:
we might want to move the warning from being inside `meta` to just being a public property on the classes below. There isn't really a need to add it to the `FormNode` interface, since there is no need for it to be passed in trough templates. This would also stop erroneously passing it trough templates.